exampleCourse/questions/demo/autograder/c/factorial/info.json

```diff
@@ -0,0 +1,14 @@
+{
+  "uuid": "c3b02ed9-3a2e-462a-9d18-f1b21cab65d1",
+  "title": "Recursive implementation of factorial",
+  "topic": "C",
+  "singleVariant": true,
+  "tags": ["jgfoster", "clang", "factorial", "c"],
```

Comment:
Should we match the tags from the other C grader example questions?

Reply:
There isn't a lot of consistency in the other C grader example questions. Anything in particular you believe this should cover?

Reply:
Most of the ones I looked at used `code` and `c`. I'd suggest dropping `clang` and `factorial`.

(The tag situation is already a mess in the example course; I'm just trying to avoid making it worse)

Reply:
I made a few consistency changes in 003303f568f09306f635be197f986d7a87e9f9c3.